apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -71,6 +72,92 @@ func TestValidateCreate(t *testing.T) {
 			oldBMH:    nil,
 			wantedErr: "hardwareRAIDVolumes and softwareRAIDVolumes can not be set at the same time",
 		},
+		{
+			name: "unsupportBMCType",
```

Comment:
Could you please add at least 1 valid and 1 invalid test cases for all validations you are adding.